freshsales-suite/contact-kyc/app/scripts/contact_entity_menu.js

```diff
@@ -0,0 +1,227 @@
+// ----------------------------
+// Setup
+// ----------------------------
+function $(el) {
+  return document.querySelector(el);
+}
+
+document.onreadystatechange = function () {
+  if (document.readyState === "interactive") {
+    app
+      .initialized()
+      .then(function (client) {
+        client.events.on("app.activated", function () {
+          onAppActivate(client);
+        });
+      })
+      .catch(handleErr);
+  }
+};
+
+async function getContact(client, currentEntityId, iparams) {
+  const url = `${iparams.fcrm_domain}api/contacts/${currentEntityId}`;
```

Comment:
Change this based on the `type` chosen for the `fcrm_domain` field in iparams.

Reply:
`url` returns as the string value itself so retained this.